cdb2api/cdb2api_priv.h

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
You don't need to do this. Just `#include "cdb2api.c"` in your unit tests.

Reply:
Well, the point is to stub out functions that any given function needs. If I include cdb2api.c, there would be duplicate function defined.
Coincidentally, in this case (get_comdb2db_hosts) I dont use hndl so i could have done without this change, but other functions will need to verify variables in it.

Reply:
Can't we have whatever is needed for unit tests in the actual cdb2api.c functions #ifdefs?